docs/index.rst

```diff
@@ -85,7 +85,7 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 | :ref:`dogpile.cache`                             | >= 0.6        | Yes            |
 +--------------------------------------------------+---------------+----------------+
-| :ref:`elasticsearch`                             | >= 1.6, < 8.0 | Yes            | 
```

Comment:
This change should be reflected in the [public corporate documentation](https://github.com/DataDog/documentation/blob/9b9442134a92f1be867987ff4fe8724b9fdc29b4/content/en/tracing/trace_collection/compatibility/python.md).

Reply:
https://github.com/DataDog/documentation/pull/20075
That's a beast of a fork to change one line (so I made it two) 😅